sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
We have v7 betas only.  I'm distinguishing between a beta and a fully released version.  People shouldn't get too attached to betas.  Tracking your app against a beta dependency is an **experiment**.  If you want to do it, cool, you can give feedback and peer into a possible future.  But there is materially no social (or SemVer) contract of what you see in that beta being preserved in a production release.  Not features, not APIs, not UI, not documentation and not dependencies.  The fact of having conducted an experiment does not induce this expectation in the project, even if it has that psychological effect on the experimenter.  There may be a dozen more betas between that version and release.  The very next beta may totally alter or erase whatever I looked at in the last one.  In fact, that project in beta may *never* see a proper release at all (e.g. indie video games, hopefully not us!).  

If we were between release candidates instead, I might feel differently, but we aren't.